dev-docs/bidders/yieldone.md

```diff
@@ -2,22 +2,23 @@
 layout: bidder
 title: YIELDONE
 description: Prebid YIELDONE Bidder Adaptor
-
 top_nav_section: dev_docs
 nav_section: reference
-
 hide: true
-
 biddercode: yieldone
-
 biddercode_longer_than_12: false
 prebid_1_0_supported : true
+media_types: banner, video
 ---
 
+### Note:
+
+THE YIELDONE adapter requires setup and approval from the YIELDONE team.
+Please reach out to your account team or y1s@platform-one.co.jp for more information.
 
 ### bid params
 
 {: .table .table-bordered .table-striped }
 | Name          | Scope    | Description      | Example | Type     |
 |---------------|----------|------------------|---------|----------|
-| `placementId` | required | The placement ID |         | `string` |
+| `placementId` | required | The placement ID | `36891` | `string` |
```

Comment:
If placementId is a string, please put it in quotes

Reply:
we fixed error.  please check it.